### PR TITLE
Set isNew to true when building editor

### LINKED ISSUE #6076: Adding widget to flowpart does not consider the IsNew context used from driver.
Adding a widget to a flow part with various fields that allow setting default values when new instance is created  (like numeric field, boolean field) is not respected.
Example:
Add a widget with a nemric field and set its default value to 100\. Then add the widget to the flow part .
The default value in editor is ignored.
This is because of 
 
https://github.com/OrchardCMS/OrchardCore/blob/acee23ef121a09568198ebda164d591ae9f084ed/src/OrchardCore.Modules/OrchardCore.Widgets/Views/ContentCard.cshtml#L22
 
setting the Isnew context variable to false.

### PR DESCRIPTION
**Pull request by giannik**
fixes  #6076

### CODE REVIEW

**Comment by hishamco** [comment]
@agriffard please merge if it's looks good to you

File: src/OrchardCore.Modules/OrchardCore.Widgets/Views/ContentCard.cshtml [hunk 1]
```diff
@@ -19,7 +19,7 @@
         }
 
         //Build Editor for Content Item
-        dynamic contentItemEditor = await ContentItemDisplayManager.BuildEditorAsync(contentItem, updater, false, "", Model.PrefixValue);
+        dynamic contentItemEditor = await ContentItemDisplayManager.BuildEditorAsync(contentItem, updater, true, "", Model.PrefixValue);
```
Comment (ns8482e) [inline src/OrchardCore.Modules/OrchardCore.Widgets/Views/ContentCard.cshtml:22]: This is for existing content and isnew should be false

Comment (deanmarcussen) [reply src/OrchardCore.Modules/OrchardCore.Widgets/Views/ContentCard.cshtml:22]: Could you look into what the correct fix is Niraj, if this is not it

Comment (ns8482e) [reply src/OrchardCore.Modules/OrchardCore.Widgets/Views/ContentCard.cshtml:22]:
```suggestion

        //AJAX request is new request and will not have CollectionShape.
        var isNew = Model.CollectionShape == null ? true : false ;
        dynamic contentItemEditor = await ContentItemDisplayManager.BuildEditorAsync(contentItem, updater, isNew, "", Model.PrefixValue);
```

Comment (ns8482e) [reply src/OrchardCore.Modules/OrchardCore.Widgets/Views/ContentCard.cshtml:22]: All new widgets are added using ajax call. In ajax call it doesn't have CollectionShape

**Comment by ns8482e** [comment]
@agriffard  @deanmarcussen little problem! suggestion is not applied yet, now its broken for existing widgets

**Comment by agriffard** [comment]
Reverted it on dev.

**Comment by deanmarcussen** [comment]
Thank you @ns8482e for the quick eyes (my bad for approving it to quickly)

Can you do another pr with the correct fix from your suggestion?

**Comment by ns8482e** [comment]
@deanmarcussen  PR created #6084

**Pull request merged**